cmd/bundle/generate/job.go

```diff
@@ -0,0 +1,109 @@
+package generate
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/cmd/root"
+	"github.com/databricks/cli/libs/cmdio"
+	"github.com/databricks/cli/libs/config"
+	"github.com/databricks/cli/libs/convert"
+	"github.com/databricks/databricks-sdk-go/service/jobs"
+	"github.com/spf13/cobra"
+)
+
+func NewGenerateJobCommand() *cobra.Command {
+	var outputDir string
+
+	cmd := &cobra.Command{
+		Use:     "job JOB_ID",
+		Short:   "Generate bundle configuration for a job",
+		Args:    cobra.MaximumNArgs(1),
+		PreRunE: root.MustConfigureBundle,
+	}
+
+	cmd.Flags().StringVarP(&outputDir, "output-dir", "d", "", `Dir path where the output config and necessary files will be stored`)
```

Comment:
This path seems like it should default to `root/resources` to fit our standard conventions for the bundle structure.

Reply:
@lennartkats-db we do a prompting for this flag later in the code with the dfault being `root/resources`